levels/sre.tex

```diff
@@ -0,0 +1,368 @@
+    a Kubernetes operator to make changes to existing infrastructure and
+    applications.
+
+    \bigbreak
+
+    May have certification like Certified Kubernetes Administrator (CKA).
+    \\ [13em]
+\hline
+    % Level
+    2
+    &
+
+    % Product Output
+    \cellcolor{blue!25}
+    Can write high quality user and product focused documentation.
+    &
+
+    % Communication
+    Provides constructive review on peers' code and design. Helps new team
+    members during their first weeks.
+    &
+
+    % Networking
+    Can independently troubleshoot most network issues.
+
+    \bigbreak
+
+    Can setup and configure production quality network infrastructure like DNS,
+    Load Balancers, and encrypted networks (like IPsec or WireGuard). Can reason
+    about their reliability.
+    &
+
+    % Security
+    Understands and can apply basic cryptographic principles. Can configure SSH
+    and TLS for a server (chooses appropriate key sizes, algorithms, and versions),
+    pick strong authentication and authorization primitives, and appropriate
+    encryption for data in transit and rest.
+
+    \bigbreak
+
+    Can setup production quality encrypted networking (like IPsec or
+    WireGuard). Can reason about their reliability.
+    &
+
+    % Systems Engineering
+    &
+
+    % SRE
+    Excellent systems troubleshooter.
+
+    \bigbreak
+
+    Familiar with and can configure alerts for system performance (CPU, memory,
+    disk, networking utilization). Uses modern BPF tools for troubleshooting.
+
+    \bigbreak
+
+    Is a member of on-call rotation and can resolve issues outlined in
+    runbooks.
+
+    \bigbreak
+
+    Demonstrates knowledge of AWS. May have certification like AWS Certified
+    SysOps Administrator.
+    \\ [13em]
+\hline
+    % Level
+    3
+    &
+
+    % Product
+    \cellcolor{blue!25}
+    Collaborates with the team to scope requirements, based on good
+    understanding of existing longer term product vision and estimates of the
+    system design of a feature of a product.
+    &
+
+    % Communication
+    Supports less experienced peers' technical skills, answering questions and
+    being a resource. Documents and improves team practices.
+    &
+
+    % Networking
+    Can setup and operate a multi-region infrastructure and networking
+    environment. Can reason about it's performance, reliability, and failure modes.
+
+    \bigbreak
+
+    Has in-depth understanding of container networking. Can write own CNI plugin
+    utilizing IPSec or WireGuard on Kubernetes.
+
+    \bigbreak
+
+    Understands advanced networking concepts such as network namespaces, NAT
+    traversal, and BPG.
+    &
+
+    % Security
+    Can build secure systems that will pass quality security audit that will
+    uncover few to no critical system design errors.
+
+    \bigbreak
+
+    Can apply security principals when building systems. Can utilize access
```

Comment:
I would move the IAM / RBAC piece, and the CloudWatch alerts, down do lvl2. These are actually fairly common skills.